ee/tuf/generator/generator.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/logutil"
+	"github.com/theupdateframework/go-tuf/client"
+	filejsonstore "github.com/theupdateframework/go-tuf/client/filejsonstore"
+)
+
+// initialRootJSON contains the trusted first version of root.json
+// This is version 1 from https://tuf.kolide.com/repository/1.root.json
+var initialRootJSON = []byte(`{"signed":{"_type":"root","spec_version":"1.0","version":1,"expires":"2023-03-29T15:57:20-04:00","keys":{"53a3eb57d9f5e755ec73e95429033b78423c9c01cef8c8d4e9911b6fe01198c8":{"keytype":"ed25519","scheme":"ed25519","keyid_hash_algorithms":["sha256","sha512"],"keyval":{"public":"344ca850ae47f65f6ffd6e9e6268524a237401000dcb892b9ddad82ca0083a65"}},"bcd5f36e7a357e7fba38b6b7c531f7fcf509eb2da2cfcdfa94bce99ad18a7317":{"keytype":"ed25519","scheme":"ed25519","keyid_hash_algorithms":["sha256","sha512"],"keyval":{"public":"8adc59a24b5c044794e002002792a4306bb63337d1f2cec961dbe68ccafdc874"}},"bd9bbc72f9a1c7cd5998c94b5a31fedb997315f81758f90dbe393743ec984508":{"keytype":"ed25519","scheme":"ed25519","keyid_hash_algorithms":["sha256","sha512"],"keyval":{"public":"aea7938b3b8f9cc0fc910899beb6a025487db87d6c2ea9a12fdf54c25817c814"}},"e434c83555ba93337e91c073095a164e4ae66f01a91cf5e8d4228f2382568d19":{"keytype":"ed25519","scheme":"ed25519","keyid_hash_algorithms":["sha256","sha512"],"keyval":{"public":"785323acb700ba835da550c1ac56300cf87890b4367c7cb1c14c1299e5e37075"}}},"roles":{"root":{"keyids":["bd9bbc72f9a1c7cd5998c94b5a31fedb997315f81758f90dbe393743ec984508"],"threshold":1},"snapshot":{"keyids":["bcd5f36e7a357e7fba38b6b7c531f7fcf509eb2da2cfcdfa94bce99ad18a7317"],"threshold":1},"targets":{"keyids":["e434c83555ba93337e91c073095a164e4ae66f01a91cf5e8d4228f2382568d19"],"threshold":1},"timestamp":{"keyids":["53a3eb57d9f5e755ec73e95429033b78423c9c01cef8c8d4e9911b6fe01198c8"],"threshold":1}},"consistent_snapshot":true},"signatures":[{"keyid":"bd9bbc72f9a1c7cd5998c94b5a31fedb997315f81758f90dbe393743ec984508","sig":"b0624815cf3b8749edc68c96ffca588ce532e9cdf8ac4db4748e260c186ecb3f975d555cf25fe5ddba418f2d3d2af4e5c2a2046b20619ddfd30f3390560f850d"}]}`)
```

Comment:
Feels weird to have this, `ee/tuf/assets/tuf/root.json` is right there. Can we call something there?

Reply:
I was wondering if we'd be able to move either `ee/tuf/assets/tuf/root.json` to a location where we are able to embed it here, or whether we'd be able to move the generator to `ee/tuf` to be able to straightforwardly embed it. Looking at it further, I don't actually think either of those options would work, unfortunately (though maybe worth confirming that just to be sure!).

I don't think it's terrible to have the duplication, either. `ee/tuf/assets/tuf/root.json` is the latest version, while this is the known/validated first version.

Reply:
I can get behind having one that's checked in and lagging, and one that's generated and used at build time. _Maybe_. 

But I don't like having 3 checked in.

Reply:
Are we able to remove `ee/tuf/assets/tuf/root.json` and the one in `packaging` from source control and add those files to our .gitignore without anything complaining? If so, sounds like that might just work. 🙂

Reply:
Looks like it's working! So we now only have one checked in. 🥳

Reply:
My feeling is that there's value in having one checked in -- using it as the start point for the build time generation, and then building in whatever this does. 

That ties the chain of trust to something in git, and gets a current version into the build

Reply:
@directionless FYI -- we ended up having to back part of this change out (https://github.com/kolide/launcher/pull/2154). When we pull in the `pkg/packaging` package in our package building tool, it expects the asset file to already exist, and we can't generate it from inside the other repository. We'll have to revisit this later to see if we can figure out how to remove that one, but this will at least let us build packages with the new changes for now!